src/components/content/home/DeployDashBoard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { Alert, Card, Col, Row, Statistic } from 'antd';
+import { Link } from 'react-router-dom';
+import { myServicesRoute } from '../../utils/constants';
+import { useQuery } from '@tanstack/react-query';
+import { ApiError, Response, ServiceService, ServiceVo } from '../../../xpanse-api/generated';
+import { convertStringArrayToUnorderedList } from '../../utils/generateUnorderedList';
+
+export function DeployDashBoard(): React.JSX.Element {
+    const [successfulDeployments, setSuccessfulDeployments] = useState<number>(0);
+    const [failedDeployments, setFailedDeployments] = useState<number>(0);
+    const [successfulDestroys, setSuccessfulDestroys] = useState<number>(0);
+    const [failedDestroys, setFailedDstroys] = useState<number>(0);
+    const [servicesLoadingError, setServicesLoadingError] = useState<React.JSX.Element>(<></>);
+    const listDeployedServicesQuery = useQuery({
```

Comment:
this query is used also in MyServices module. We must move this to one single place and use it from both places.